docs/api/sql/Raster-operators.md

```diff
@@ -692,6 +692,9 @@ In order to remove an existing noDataValue from an existing band, pass null as t
 
 Note that: `bandIndex == RS_NumBands(raster) + 1` is an experimental feature and might not lead to the loss of raster metadata and properties such as color models.
 
+!!!Note
+    RS_BandFromArray typecasts the double band values to the given datatype of the raster. This can lead to overflow values if values beyond the range of the raster's datatype are provided.
```

Comment:
This should be `RS_AddBandFromArray`. What can users do to find the current data type in the raster?

Reply:
Pushed the mistake fix.
To get the data type for the given band, users can use RS_BandPixelType